src/constants_32bit.rs

```diff
@@ -71,35 +51,18 @@ pub const MSQRT_M1: FieldElement32 = FieldElement32([
     32595792,    7943725,  -9377950,  -3500415, -12389472,
     272473,     25146209,   2005654,   -326686, -11406482, ]);
 
-/// Precomputed value of 1/2 (mod p).
-pub const HALF: FieldElement32 = FieldElement32([
-    10, 0, 0, 0, 0, 0, 0, 0, 0, -16777216, ]);
-
 /// In Montgomery form y² = x³+Ax²+x, Curve25519 has A=486662.
 pub const A: FieldElement32       = FieldElement32([
     486662, 0, 0, 0, 0, 0, 0, 0, 0, 0, ]);
 
 /// `APLUS2_OVER_FOUR` is (A+2)/4. (This is used internally within the Montgomery ladder.)
 pub const APLUS2_OVER_FOUR: FieldElement32 = FieldElement32([121666, 0, 0, 0, 0, 0, 0, 0, 0, 0]);
 
-/// `SQRT_MINUS_A` is sqrt(-486662)
-// XXX I think that this was used in Adam's code for his elligator
-// implementation, but that should maybe be using sqrt(-486664)
-// instead...?  - hdevalence
-pub const SQRT_MINUS_A: FieldElement32 = FieldElement32([ // sqrtMinusA
-    12222970,    8312128,  11511410,  -9067497,  15300785,
-    241793,    -25456130, -14121551,  12187136,  -3972024, ]);
-
```

Comment:
This is used in the elligator2 implementation which matches AGL's golang implementation (albeit none of the others, since all of them do different things).

I guess I should add it back in if/when we merge the elligator2 implementations?